### PR TITLE
Add datasource/JPA integration, edit swagger v4 spec

### PR DESCRIPTION
**Pull request by igordumchykov**

### CODE REVIEW

File: src/main/java/com/icthh/xm/tmf/ms/prepaybalance/config/Constants.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.icthh.xm.tmf.ms.prepaybalance.config;
+
+/**
+ * Application constants.
+ */
+public final class Constants {
+
+    public static final String SYSTEM_ACCOUNT = "system";
+
+    private Constants() {
```
Comment (VictorKhoroshko) [inline src/main/java/com/icthh/xm/tmf/ms/prepaybalance/config/Constants.java]: You can use lombok @UtilityClass

File: src/main/resources/config/application-prod.yml [hunk 2]
```diff
@@ -37,6 +37,34 @@ spring:
...
+  datasource:
+    type: com.zaxxer.hikari.HikariDataSource
+    url: jdbc:oracle:thin:@localhost:1521:xe
+    username: system
+    password: oracle
+    hikari:
+      poolName: Hikari
+      auto-commit: false
+  jpa:
+    database-platform: org.hibernate.dialect.Oracle12cDialect
```
Comment (VictorKhoroshko) [inline src/main/resources/config/application-prod.yml]: AFAIK XM uses postgres db by default. It looks like this customization is for MW only.

File: src/main/resources/config/application-dev.yml [hunk 3]
```diff
@@ -41,6 +41,27 @@ spring:
         prefer-ip-address: true
       host: localhost
       port: 8500
+  datasource:
+    type: com.zaxxer.hikari.HikariDataSource
+    url: jdbc:h2:mem:loyalty;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: loyalty
```
Comment (yepashkov) [inline src/main/resources/config/application-dev.yml]: should we change username?

File: src/main/java/com/icthh/xm/tmf/ms/prepaybalance/config/Constants.java (hunk 1 above)
Comment (yepashkov) [inline src/main/java/com/icthh/xm/tmf/ms/prepaybalance/config/Constants.java]: probably https://projectlombok.org/features/experimental/UtilityClass ?

File: src/main/resources/config/application-dev.yml (hunk 3 above)
Comment (igordumchykov) [reply src/main/resources/config/application-dev.yml]: Done

File: src/main/java/com/icthh/xm/tmf/ms/prepaybalance/config/Constants.java (hunk 1 above)
Comment (igordumchykov) [reply src/main/java/com/icthh/xm/tmf/ms/prepaybalance/config/Constants.java]: Done

**Review by yepashkov — APPROVED** [review]

**Review by VictorKhoroshko — APPROVED** [review]

File: src/main/resources/config/application-prod.yml (hunk 2 above)
Comment (igordumchykov) [reply src/main/resources/config/application-prod.yml]: Changed, done

**Pull request merged**